app/src/main/java/happy/sopt/samsunghealth/main/home/HomeFragment.kt

```diff
@@ -1,23 +1,48 @@
 package happy.sopt.samsunghealth.main.home
 
+import android.content.Intent
 import android.os.Bundle
 import android.view.LayoutInflater
+import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
+import happy.sopt.samsunghealth.R
 import happy.sopt.samsunghealth.databinding.FragmentHomeBinding
+import happy.sopt.samsunghealth.record.RecordActivity
 
 class HomeFragment : Fragment() {
     private var _binding: FragmentHomeBinding? = null
     private val binding: FragmentHomeBinding get() = _binding!!
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?) =
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?) =
         FragmentHomeBinding.inflate(inflater, container, false).let {
-            _binding = it
-            binding.root
+        _binding = it
+        binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        clickEvent()
+    }
+
+    private fun clickEvent() {
+        binding.btInputweight.setOnClickListener {
```

Comment:
```suggestion
        binding.btInputWeight.setOnClickListener {
```
`xml` 파일에서 `bt_input_weight` 처럼 만들면 뷰바인딩에서 알아서 snake_case를 camelCase로 만들어줍니다.
xml에선 snake_case로 써도 됩니다. 보통 그렇게 하기도 하고요